README.md

```diff
@@ -106,23 +106,91 @@ and [Kopytko Unit Testing Framework](https://github.com/getndazn/kopytko-unit-te
 
 Enforces a proper alphabetical and path-specific order of `@import` and `@mock` annotations:
 - `@mock` annotations have to be after all `@import` annotations
+- imports or mocks from external packages have to be before local ones
+- alphabetical order of external packages names
 - alphabetical order of paths
 - nested paths have to be declared after their root path
 
-Examples of **incorrect** code for this rule:
+#### Imports before mocks
+
+Example of **incorrect** code:
 ```brightscript
 ' @mock /components/mocked-function.brs
+' @import /components/main-function.brs
+' @import /components/nested/another-function.brs
 ' @import /components/nested/cool-function.brs
+```
+
+Example of **correct** code:
+```brightscript
+' @import /components/main-function.brs
 ' @import /components/nested/another-function.brs
+' @import /components/nested/cool-function.brs
+' @mock /components/mocked-function.brs
+```
+
+#### External packages before local
+
+Example of **incorrect** code:
+```brightscript
 ' @import /components/main-function.brs
+' @import /components/nested/another-function.brs
+' @import /components/nested/cool-function.brs
+' @import /components/nested/some-function.brs from package-name
+' @mock /components/mocked-function.brs
+' @mock /components/some-mocked-function.brs from package-name
 ```
 
-Examples of **correct** code for this rule:
+Example of **correct** code:
+```brightscript
+' @import /components/nested/some-function.brs from package-name
+' @import /components/main-function.brs
+' @import /components/nested/another-function.brs
+' @import /components/nested/cool-function.brs
+' @mock /components/some-mocked-function.brs from package-name
+' @mock /components/mocked-function.brs
+```
+
+#### Alphabetical order of external packages names
+
+Example of **incorrect** code:
+```brightscript
+' @import /components/a-function.brs from package-name
+' @import /components/some-function.brs from another-package-name
+' @import /components/main-function.brs
+' @mock /components/another-mocked-function.brs from package-name
+' @mock /components/some-mocked-function.brs from another-package-name
+' @mock /components/mocked-function.brs
+```
+
+Example of **correct** code:
+```brightscript
+' @import /components/some-function.brs from another-package-name
+' @import /components/a-function.brs from package-name
+' @import /components/main-function.brs
+' @mock /components/some-mocked-function.brs from another-package-name
+' @mock /components/another-mocked-function.brs from package-name
+' @mock /components/mocked-function.brs
```

Comment:
Wondering if it wouldn't be better to have external imports first, then the rest. What do you think?

Reply:
yeah, maybe it would be even more readable?

Reply:
Actually, I am not sure. Probably this is a matter of subjective preference.

Reply:
or maybe add two rules, and anyone can use what suits them better?

Reply:
If we change it will we have to fix all the occurances?

Reply:
depends on the implementation, I won't add it here, so we need to decide which way we want to go